hapi-loader/src/main/java/com/arkhn/hapi/loader/AutoTimingConfiguration.java

```diff
@@ -0,0 +1,18 @@
+package com.arkhn.hapi.loader;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.EnableAspectJAutoProxy;
+
+import io.micrometer.core.aop.TimedAspect;
+import io.micrometer.core.instrument.MeterRegistry;
+
+@Configuration
+@EnableAspectJAutoProxy
+
+public class AutoTimingConfiguration {
```

Comment:
Could you add a comment to tell what this class is here for, please? 🙏

Reply:
done!